src/less/contact.less

```diff
@@ -0,0 +1,112 @@
+.contact {
+    display: flex;
+    justify-content: space-between;
+}
+
+.form {
+    border: 1px solid black;
+    padding: 25px;
+    flex-basis: 45%;
+}
+
+.form-email-section {
+    display: flex;
+    align-items: baseline;
+}
+
+.form-email-section__title {
+    margin-right: 5px;
+}
+
+.title__email-input {
+    flex-grow: 1;
+    border: 0;
+    outline: 0;
+    border-bottom: 1px solid black;
+    padding-left: 20px;
+    //.u-svg-fill('../picture/email.svg', @light-blue);
+}
+
+.form-message-title {
+    margin-bottom: 0;
+}
+
+.form-message-title-section {
+    display: flex;
+}
+
+.title-section__text-area {
```

Comment:
title-section is to general you have to specify it: form-title, tour-title, page-title....
one of the main role of the css class names are to be readable, if I see a class name I should be able to identify the html element that has the class name only based on the class